docs/docs/working-with-images.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Working With Images In Gatsby
+---
+
+# Working With Images In Gatsby
+
+Gatsby has several useful [plugins](/docs/plugins/) that work together for adding and optimizing images for [page components](/docs/building-with-components/#page-components)
+
+
+The recommend approach is to 
+* GraphQL to query image files
+* Sharp, an image processing library, to transform images
+* The Gat
+
+
+## Access Your Images With GraphQL
+
+[GraphQL can access image files](docs/querying-with-graphql/#images) with the [`gatsby-source-filesystem`](/packages/gatsby-source-filesystem/) plugin. 
+
+Once you have this installed you should be able to query the files as shown in the our tutorial on [writing queries](tutorial/part-four/#build-a-page-with-a-graphql-query). 
+
+
+
+## Formatting Images With Sharp
+
+[Sharp](https://github.com/lovell/sharp) is a high-performance image processing library. The [`gatsby-transformer-sharp`](/packages/gatsby-transformer-sharp/) and [`gatsby-plugin-sharp`](/packages/gatsby-plugin-sharp) plugins allow GraphQL queries to not only query images, but transform them.
+
+
+Common transformations include `size` and `resolution` which are key for optimizing images. See [tocumentation for `gatsby-plugin-sharp`](/packages/gatsby-plugin-sharp/) to view all the transformation methods you can use.
+
+This also gives you access to special queries called [fragments](/packages/gatsby-image/#fragments) that automatically query a bunch of field's we'll want to use later including width, height, and src. 
+
+## Optimizing Images With Gatsby Image
+
+[`gatsby-image`](/packages/gatsby-image/) is a plugin that automatically creates React components for optimized images that:
+
+
+> * Loads the optimal size of image for each device size and screen resolution
+> * Holds the image position while loading so your page doesn't jump around as images load
+> * Uses the "blur-up" effect i.e. it loads a tiny version of the image to show while the full image is loading
+> * Alternatively provides a "traced placeholder" SVG of the image.
+> * Lazy loads images which reduces bandwidth and speeds the initial load time
+> * Uses [WebP](https://developers.google.com/speed/webp/) images if browser supports the format
+
+### Chosing The Right Approach
+
+Which methods and fragments you use will depend on what type of responsive image you have. See the [Gatsby Image documentation for more information](/packages/gatsby-image/#two-types-of-responsive-images) . In general for fixed size images use resolution, for images you want to stretch use size. 
```

Comment:
In this last sentence it seems like you are referring to the `gatsby-transformer-sharp` plugin. Is that correct?

If so, I'd use this document as more of a "here's an overview of how to do cool stuff with images in Gatsby and a quick comparison of these two main plugins" and then place any code examples that are specific to `gatsby-transformer-sharp` in whatever READme and/or doc associated with that plugin.

If you are _not_ referring to the `gatsby-transformer-sharp` plugin and the last section is more general information, I guess I'm lost because the explanation about `gatsby-image` didn't mention anything about resolution.